daml-script/export/src/main/scala/com/daml/script/export/TreeUtils.scala

```diff
@@ -465,26 +468,59 @@ object TreeUtils {
     case Kind.Exercised(value) => valueRefs(value.getChoiceArgument.sum)
   }
 
+  /** All identifiers referenced by the given value that require a module import.
+    *
+    * Note, the package-id component of the identifier may be empty if there is no
+    * corresponding package-id available. E.g. DA.Time.Time incurred by Timestamp values.
+    */
   def valueRefs(v: Value.Sum): Set[Identifier] = v match {
     case Sum.Empty => Set()
     case Sum.Record(value) =>
-      Set(value.getRecordId).union(value.fields.foldMap(f => valueRefs(f.getValue.sum)))
+      val fieldRefs = value.fields.foldMap(f => valueRefs(f.getValue.sum))
+      if (isTupleId(value.getRecordId)) {
+        fieldRefs
+      } else {
+        Set(value.getRecordId).union(fieldRefs)
```

Comment:
```suggestion
        fieldRefs + value.getRecordId
```
feel free to ignore but maybe a bit clearer and probably slightly more efficient (I highly doubt that matters).

Reply:
If you don’t like the operator there’s `.incl`.

Reply:
Good point, this was copy-pasta from some of the other cases. I've adjusted them to use `+` consistently.